clinica/iotools/converters/genfi_to_bids/genfi_to_bids_utils.py

```diff
@@ -75,6 +82,20 @@ def filter_dicoms(df: DataFrame) -> DataFrame:
     return df
 
 
+def _handle_series_description(x: str) -> str:
+    series_description = pdcm.dcmread(x).SeriesDescription
+    if any([modality in series_description for modality in ["t1", "t2", "T1", "T2"]]):
+        echo_time = pdcm.dcmread(x).EchoTime
+        repetition_time = pdcm.dcmread(x).RepetitionTime
+        if (
+            any([modality in series_description for modality in ["t2", "T2"]])
+            and echo_time < 60
```

Comment:
You explained these magic numbers in the PR but not in the code such that they will still seem pretty magical in a few months/years for someone reading this...

I would both use constant with explicit names (`MAX_ECHO_TIME_FOR_A_T1` and `MAX_REPETITION_TIME_FOR_A_T1` maybe???) and put the explanations you wrote in the PR in a docstring of this function.